src/physics/lsm_driver.f90

```diff
@@ -451,9 +475,9 @@ subroutine lsm_init(domain,options)
         implicit none
         type(domain_t), intent(inout) :: domain
         type(options_t),intent(in)    :: options
-        integer :: i
+        integer :: i,j
 
-        if (options%physics%landsurface == 0) return
+        if (options%physics%landsurface == 0) return   !! So we cannot (currently) run without lsm but with water.
```

Comment:
good point, at some point it might make sense to pull this out of the "lsm_driver" code...

Reply:
yes, although the scenario (lsm=0, water=3) is a bit unlikely, except for debugging purposes maybe.